pubsub/test/iam.test.js

```diff
@@ -20,18 +20,19 @@ var policyMock = 'policy';
 var permissionsMock = 'permissions';
 
 function getSample () {
+  var apiResponseMock = {};
   var subscriptionMock = {
     iam: {
       getPolicy: sinon.stub().callsArgWith(0, null, policyMock),
```

Comment:
`iam.getPolicy` does not return an `apiResponse`, contrary to what the gcloud-node docs say. Similar issue with `getMetadata`.